pyproject.toml

```
@@ -0,0 +1,9 @@
+[build-system]
+requires = [
+    "build",
+    "setuptools>=64",
+    "torch",
+    "wheel"
```

If you depend on setuptools 70.1 or later, you won't need `wheel`.
```suggestion
    "setuptools>=70.1",
    "torch"
```